ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+import java.util.Random;
+public class ColorSupplier {
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);

```

Comment:
random should be class field - now you're recreating it per each method invocation